sigsep_open-unmix-pytorch_umx.md

```diff
@@ -13,6 +13,7 @@ github-id: sigsep/open-unmix-pytorch
 featured_image_1: sigsep_umx-diagram.png
 featured_image_2: no-image
 accelerator: cuda-optional
+demo-model-link: https://huggingface.co/spaces/pytorch/open-unmix
```

Comment:
I think this one doesn't use torch.hub.load and instead has a harcoded dependency on `openunmix`, is this on purpose?

Reply:
updated the space to use `torch.hub.load`, I was following the colab in the repo in this case to use `predict.seperate` previously